packages/vitaminjs-runtime/package.json

```diff
@@ -16,6 +16,9 @@
     "type": "git",
     "url": "https://github.com/Evaneos/vitaminjs.git"
   },
+  "bin": {
+    "vitamin-serve": "bin/vitamin-serve.js"
```

Comment:
as discussed with @mdarse we may consider `serve` task as part of vitamin-build instead. Let's talk

Reply:
It has been indeed removed from `vitamin-runtime`. The package.json has a mistake :)